lib/taskana-core/src/test/java/acceptance/task/UpdateManualPriorityWithSpiAccTest.java

```diff
@@ -0,0 +1,143 @@
+package acceptance.task;
+
+import static acceptance.DefaultTestEntities.defaultTestClassification;
+import static acceptance.DefaultTestEntities.defaultTestObjectReference;
+import static acceptance.DefaultTestEntities.defaultTestWorkbasket;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import acceptance.priorityservice.TestStaticValuePriorityServiceProvider;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import testapi.TaskanaInject;
+import testapi.TaskanaIntegrationTest;
+import testapi.WithServiceProvider;
+
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.classification.api.models.ClassificationSummary;
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.test.security.WithAccessId;
+import pro.taskana.spi.priority.api.PriorityServiceProvider;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.models.ObjectReference;
+import pro.taskana.task.api.models.Task;
+import pro.taskana.task.internal.builder.TaskBuilder;
+import pro.taskana.task.internal.jobs.helper.TaskUpdatePriorityWorker;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.api.models.WorkbasketSummary;
+import pro.taskana.workbasket.internal.builder.WorkbasketAccessItemBuilder;
+
+@TaskanaIntegrationTest
+@WithServiceProvider(
+    serviceProviderInterface = PriorityServiceProvider.class,
+    serviceProviders = TestStaticValuePriorityServiceProvider.class)
+class UpdateManualPriorityWithSpiAccTest {
+
+  @TaskanaInject TaskanaEngine taskanaEngine;
+  @TaskanaInject TaskService taskService;
+  @TaskanaInject ClassificationService classificationService;
+  @TaskanaInject WorkbasketService workbasketService;
+
+  ClassificationSummary defaultClassificationSummary;
+  WorkbasketSummary defaultWorkbasketSummary;
+  ObjectReference defaultObjectReference;
+  TaskUpdatePriorityWorker worker;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup() throws Exception {
+    worker = new TaskUpdatePriorityWorker(taskanaEngine);
+    defaultClassificationSummary =
+        defaultTestClassification().priority(9).buildAndStoreAsSummary(classificationService);
+    defaultWorkbasketSummary = defaultTestWorkbasket().buildAndStoreAsSummary(workbasketService);
+
```

Comment:
See comments from previous file:
Either use `asSummary` or remove the `...Summary` from the instance variables